### DISCUSSION TITLE
Any specific use case for supporting only VARCHAR datatype and not NVARCHAR

### DISCUSSION THREAD

**Discussion by ade-owasp-sf** [milvus-io/milvus#22871] [Ideas & Feature requests]
Any specific use case for supporting only VARCHAR datatype and not NVARCHAR

With NVARCHAR there will be less code page issues.

**Comment by yhmo** [comment]
Agree, it is valuable for Unicode strings.

**Comment by Strengthless** [comment]
Bump, is there any solution to this yet?

**Reply by yhmo** [reply]
> NVARCHAR

Not yet. I suppose NVARCHAR is a new data type of field. So far we haven't put it in our plan.
